content/300-guides/100-performance-and-optimization/150-connection-management/200-configure-for-external-connection-pooler.mdx

```diff
@@ -1,38 +1,44 @@
 ---
-title: Configure Prisma Client with PgBouncer
+title: Configure Prisma Client with an external connection pooler (like PgBouncer)
```

Comment:
```suggestion
title: Configure Prisma Client with an external connection pooler
```

I don't think we need it in the title but we could include it as part of the metaTitle and metaDescription.

Reply:
Let's keep it so Google does not get weird ideas to rank other pages from ours for `PgBouncer`. Title has the biggest effect on that.